### ISSUE TITLE
For different number of classes?

### ISSUE THREAD

**Issue by AI-Hunter** [yanx27/Pointnet_Pointnet2_pytorch#153]
Can it work for different number of classes?
My class number = 2
Where I change in train_semseg.py? 
What is label_weights,  the value of the similar to class number?
Anyone help me please.

**Comment by LucHayward** [comment]
Yes it does work for arbitrary class numbers (though these must exist in your training data).

You need to change the final layers output size in the model definition.
